components/screens/Send.tsx

```diff
@@ -21,19 +21,26 @@ export interface SendFormData {
   domain: string | undefined;
 }
 
+export interface State {
+  form?: SendFormData;
+  step?: Step;
+}
```

Comment:
It allows ledger send can move the confirmation page by having step data when using `useLocation`.